public/javascripts/deer/components/view/view.js

```diff
@@ -0,0 +1,75 @@
+import { default as UTILS } from '../../deer-utils.js'
+import { default as DEER } from '../../deer-config.js'
+
+const template = (obj, options = {}) => {
+    let indent = options.indent ?? 4
+    let replacer = (k, v) => {
+        if (DEER.SUPPRESS.indexOf(k) !== -1) return
+        return v
+    }
+    try {
+        return `<pre>${JSON.stringify(obj, replacer, indent)}</pre>`
+    } catch (err) {
+        return null
+    }
+}
+
+export default class DeerView extends HTMLElement {
+    static get observedAttributes() { return [`${DEER.PREFIX}-id`,`${DEER.PREFIX}-final`,`${DEER.PREFIX}-lazy`,`${DEER.PREFIX}-key`,`${DEER.PREFIX}-list`,`${DEER.PREFIX}-link`,`${DEER.PREFIX}-listening`]; }
+    #final = false
+
+    constructor() {
+        super()
+        this.template = DEER.TEMPLATES[this.getAttribute(`${DEER.PREFIX}-template`)] ?? template
+    }
+
+    connectedCallback() {
+        this.innerHTML = this.innerHTML?.trim() ?? `<small>&copy;2022 Research Computing Group</small>`
+        UTILS.worker.addEventListener('message', e => {
+            if (e.data.id !== this.getAttribute(`${DEER.PREFIX}-${DEER.ID}`)) { return }
+            switch (e.data.action) {
+                case "update":
+                    this.innerHTML = this.template(e.data.payload)
```

Comment:
This is a template issue, rather than a DEER integration issue. In this case, if any object comes back it tries to draw, so the default loader is overwritten when an `id`-only Object is passed in. It shouldn't, but that's not a fault here.

Reply:
#18 ... also my previous comment vaporized and may come back, but it is a template issue, in short.